src/sentry/snuba/metrics/extraction.py

```diff
@@ -81,6 +81,7 @@
     "geo.subdivision": "user.geo.subdivision",
     "http.method": "request.method",
     "http.url": "request.url",
+    "http.referer": "request.referer",
```

Comment:
The spec was failing to be generated with this error:
https://github.com/getsentry/sentry/blob/f1baa3e033998a570ecb082f97b19f67b21b288a/src/sentry/snuba/metrics/extraction.py#L1292

This would make it fallback to Discover.